index.html

```diff
@@ -669,6 +669,30 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ========= Card 12 END =========  -->
+        <div class="card">
+          <p class="name">Vera</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">tschanzvera</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">tschanzvera</a>
```

Comment:
Kindly add up your profile link in the `href` attribute. Replace the `your_user_handle` so that the link redirects to your profile. An example could be as follows. 
```
https://www.github.com/tschanzvera
```

Reply:
correctly configured github and changed twitter to linkedin